test/parallel/test-util-inspect.js

```diff
@@ -288,17 +288,21 @@ assert.equal(util.inspect(setter, true), '{ [b]: [Setter] }');
 assert.equal(util.inspect(getterAndSetter, true), '{ [c]: [Getter/Setter] }');
 
 // exceptions should print the error message, not '{}'
-assert.equal(util.inspect(new Error()), '[Error]');
-assert.equal(util.inspect(new Error('FAIL')), '[Error: FAIL]');
-assert.equal(util.inspect(new TypeError('FAIL')), '[TypeError: FAIL]');
-assert.equal(util.inspect(new SyntaxError('FAIL')), '[SyntaxError: FAIL]');
+var errors = [];
```

Comment:
@mscdex ... you're correct. `const` variables can be mutated. in this instance, tho, within the test, whether it's a `var` or `const` doesn't make much difference. More a style choice.